public/app/features/dashboard-scene/panel-edit/VizPanelManager.tsx

```diff
@@ -222,13 +214,6 @@ export class VizPanelManager extends SceneObjectBase<VizPanelManagerState> {
       newFieldConfig = restoreCustomOverrideRules(newFieldConfig, cachedFieldConfig);
```

Comment:
just test/verify that standard field config is preserved when changing plugins (unit, decimals, min/max etc)